### ISSUE TITLE
Parent Child Relations and path

### ISSUE THREAD

**Issue by yasirhaleem** [VirtusLab-Open-Source/strapi-plugin-navigation#367]
Thanks for the awesome pugin

i want to build following structure for menu using this plugin

- Top Most Entry act as wrapper (not linked to any page just a title ) 
   --Parent Item (type internal Parent Page )
      -- child item  ( is it possible when creating this item it shows only child items ? )

Other problem is how i can remove `/#/parent-page` /#` from it

<img width="1228" alt="Screen Shot 2023-09-01 at 3 13 13 AM" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/989956/26f9523f-88ec-4c4a-946d-80f5e8025d83">

**Comment by CodeVoyager** [comment]
Hi @yasirhaleem 

Can you rephrase your question, please? I'm not sure which part you want to omit.

**Comment by yasirhaleem** [comment]
Hi @CodeVoyager 

i want to omit the # part e.g

For the wrapper element i don't want it to point to any link but just text so i used '#' in url

but the child element for that wrapper element is show link as /#/linktosomepage ( can we ommit that `/#` part ? ),  if we add another nested item url becomes like this

/#/parent-item/child-item can it be like /parent-item/child-item

**Comment by yasirhaleem** [comment]
Hi @CodeVoyager 

i want to omit the # part e.g

For the wrapper element i don't want it to point to any link but just text so i used '#' in url

but the child element for that wrapper element is show link as /#/linktosomepage ( can we ommit that `/#` part ? ),  if we add another nested item url becomes like this

/#/parent-item/child-item can it be like /parent-item/child-item

**Comment by CodeVoyager** [comment]
Hi @yasirhaleem 

Currently you cannot ignore navigation items in your navigation result.

But you can achieve somewhat similar result by following configuration.

Parent/wrapper element's path is `/`. Child does not start with `/`. You just specify rest like `parent/`. And so on.

<img width="931" alt="image" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/13495487/bd30a3ed-2e40-48ec-9724-fb0eb4bc6b76">

<img width="931" alt="image" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/13495487/e66dd33e-8a52-4169-a8aa-d9a7e0e84bfc">

<img width="931" alt="image" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/13495487/4d12b058-52a3-4d37-a135-66eac3ecd28d">

Results with:

<img width="1024" alt="image" src="https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/13495487/e0217c95-7972-4409-85c1-9d2f0e642158">

If any navigation nodes are to be filtered out API's client needs to perform filtering by themself.

Hope this helps you out.

**Comment by yasirhaleem** [comment]
Hi @CodeVoyager Thanks for the example.

As per the your screenshot we can't have `#` for root level element ? Problem is i want root level menu too go no where #nogo just act as label with dropdown menu

Other solution can be to loop through each menu and replace '/#' with '/`

**Comment by CodeVoyager** [comment]
Hi @yasirhaleem 

That's right. This plugin was design to render navigation as it was designed. For your case you could on UI check only root element of your navigation and change it from `/` to `#`. To recognise root you could use `uiRouterKey` value. Or structure. Depends on mode you render your navigation in.

**Comment by CodeVoyager** [comment]
Closing due to inactivity. Re-open should you need more help.
